### PR TITLE
Use Angular to refresh results in background. 

### PR DESCRIPTION
**Pull request by OneCricketeer**
No longer uses hard-page refreshes. Fixes #77.

### CODE REVIEW

File: elasticsearch-twitter-webapp/app/index.html [hunk 1]
```diff
@@ -22,15 +22,16 @@
...
-<script src="app.js"></script>
+<script src="bower_components/elasticsearch/elasticsearch.angular.min.js"></script>
+<script src="bower_components/tweetparser.js/dist/jquery.tweetParser.min.js"></script>
+
 <script src="search/search.js"></script>
+<script src="app.js"></script>
 </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (risdenk) [inline elasticsearch-twitter-webapp/app/index.html]: Don't reorder.

File: elasticsearch-twitter-webapp/app/search/search.js [hunk 2]
```diff
@@ -4,14 +4,28 @@ angular.module('demo.search', ['ngRoute', 'elasticui'])
...
+        $scope.getData = function(){
+          $scope.indexVM.refresh();
+        };
+
+        // Function to replicate setInterval using $timeout service.
+        $scope.intervalFunction = function(){
+          $timeout(function() {
+            $scope.getData();
+            $scope.intervalFunction();
+          }, 9000);
```
Comment (risdenk) [inline elasticsearch-twitter-webapp/app/search/search.js]: is 9 seconds correct? how about 5? or 10?

Comment (OneCricketeer) [inline elasticsearch-twitter-webapp/app/search/search.js]: It was 9 before. A multiple of 5 does seem more reasonable, though. Any preference?

Comment (risdenk) [inline elasticsearch-twitter-webapp/app/search/search.js]: 10 seconds is probably fine. 5 seems to be really quick.

**Comment by OneCricketeer** [comment]
@risdenk - Good?

**Comment by risdenk** [comment]
Can you squash your commits?

**Comment by OneCricketeer** [comment]
:ok_hand:

**Comment by risdenk** [comment]
:+1:

**Pull request merged**